TypeCobol/Compiler/Diagnostics/TypeCobolChecker.cs

```diff
@@ -138,7 +138,7 @@ public static void OnNode(Node node)
                         //If there is more than one variable with the same name, it's ambiguous
                         message = string.Format("Call to '{0}'(no arguments) is ambigous. '{0}' is defined {1} times",
                             functionName,
-                            potentialVariables.Count() + functionDeclarations.Count);
+                            potentialVariablesCount + functionDeclarations.Count);
```

Comment:
`functionDeclarations.Count` is evaluated many times, it could be stored in a local variable like `potentialVariablesCount`

Reply:
`functionDeclarations` is a `List`, calling the `Count` property is inexpensive. Contrary to LINQ `Count()` method which triggers an enumeration.